src/app/views/record/record.html

```diff
@@ -2,12 +2,37 @@
   <md-progress-circular md-mode="indeterminate"></md-progress-circular>
 </div>
 <div layout="column" class="record-detail" ng-show="!$ctrl.loading">
-  <h3>{{ $ctrl.record.entity }} {{ $ctrl.getIdentifier($ctrl.record) }}</h3>
+  <h3>
+    {{ $ctrl.record.entity }} {{ $ctrl.getIdentifier($ctrl.record) }}
+    <span
+      ng-if="$ctrl.invalidUrl && $ctrl.record.entity === 'Sample_Photo'"
+      class="alert alert-danger"
+      role="alert"
+    >
+      <i class="fa fa-exclamation-circle"></i>
+      Original Url is invalid
+    </span>
+    <span
+      ng-if="$ctrl.record.processed === 'false' && $ctrl.record.entity === 'Sample_Photo'"
+      class="alert alert-danger"
+      class="text-center"
+      role="alert"
+    >
+      <i class="fa fa-exclamation-circle"></i>
+      Image is still being processed
+    </span>
+  </h3>
 
   <div class="main-details" layout="row" layout-align="center center">
     <div ng-if="$ctrl.record.img1024" flex="100">
       <a href="{{ $ctrl.record.img1024 }}"><img ng-src="{{ $ctrl.record.img512 }}"/></a>
     </div>
+    <div
```

Comment:
this section is not needed. We don't want to support base64 encoded urls for the image.

I made a bug fix in the backed that should prevent this from showing 'processed=true' and specify an error instead.

this entire section can be removed

Reply:
Ok, I'm confused on this one. I'm testing on photos that are already loaded, and removing any of  this section makes images not appear, and still shows 'processed=true'

Reply:
that's b/c those photos we loaded before I fixed a bug in the photo processing. Try and load those again and it should have an error

Reply:
Ok, got rid of the duplicate. all of my testers were base64 which is why i couldnt see the img512 before